testingt.go

```diff
@@ -0,0 +1,204 @@
+package godog
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+)
+
```

Comment:
Proposal: enforce compatibility

```suggestion

var (
	_ TestingT = (*testing.B)(nil)
	_ TestingT = (*testing.F)(nil)
	_ TestingT = (*testing.T)(nil)
)

```

Reply:
Nice pattern, I like it! Can't hurt to get a compile error if they change those elements upstream (as highly unlikely as that may be!).